### DISCUSSION TITLE
Vim Mode Plugin?

### DISCUSSION THREAD

**Discussion by pingu1m** [mdx-editor/editor#623] [General]
Hi, I currently have an app with vccode editor and I have vim mode in it. I wanted to replace with this editor and focus only on markdown. However, I want to know if there is a vim mode implemented? 

Thanks

**Comment by alexryanterry** [comment]
This would be so sweet... I'm trying to move our company to a web-based solution for shared coumentation using this react component, but I'm a heavy vim user and the idea of not having vim for my main interface into writing docs is a sad thing..

**Comment by petyosi** [comment]
https://github.com/mdx-editor/editor/issues/514 - happy to accept a PR.

**Reply by alexryanterry** [reply]
Sweet, don't have bandwidth to invest in this right now, but will come back to it when I have time. Definitely interested in contributing. I love your product. 

I did check Lexical GH / Discord and I didn't see any solutions / big discussion on adding vim to the lexical , so it doesn't seem "solved" over there. 

I did find this: https://www.npmjs.com/package/react-vim-wasm which maybe could be plugged in somehow even if it isn't an officially supported pattern. Do you have any thoughts on how that could be plugged in? I'd be happy to prove it out and create some docs / how-to if others are interested.
